### DISCUSSION TITLE
Action environment `with` should be `within`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#10774] [Actions] [answered]
This message was reported in https://github.com/actions/runner/issues/464#issue-611484210

It appears something like:
![image](https://user-images.githubusercontent.com/2119212/151706921-67515701-3241-4f52-8bfd-ccc46bd89640.png)

Here's an imaginary diff that would change the message from nonsensical to something else:
```diff
-Runner `agent name` did not respond to a cancelation request with 00:05:00.
+Runner `agent name` did not respond to a cancelation request within 00:05:00.
```

**Comment by ethomson** [comment, marked as answer]
Thanks, definitely a bug.  Appreciate the report.
